### PR TITLE
No massage

### PR DESCRIPTION
**Pull request by Antiselekotin**

### CODE REVIEW

**Comment by DzyubSpirit** [comment]
:)
Screenshot from 2019-10-19 16-01-20

File: Exercises/1-random.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-const random = (min, max) => {
-  // Generate random Number between from min to max
-  // Use Math.random() and Math.floor()
-  // See documentation at MDN
-};
+const random = (superPuperMin, superPuperMax) =>
+  // eslint-disable-next-line max-len
+  Math.floor(Math.random() * (superPuperMax - superPuperMin)) + superPuperMin;
 
+console.log(random(10, 10));
```
Comment (tshemsedinov) [inline Exercises/1-random.js]: Don't call functions, you have a tests for that

File: Exercises/1-random.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
 'use strict';
 
-const random = (min, max) => {
-  // Generate random Number between from min to max
-  // Use Math.random() and Math.floor()
-  // See documentation at MDN
-};
+const random = (superPuperMin, superPuperMax) =>
+  // eslint-disable-next-line max-len
+  Math.floor(Math.random() * (superPuperMax - superPuperMin)) + superPuperMin;
```
Comment (tshemsedinov) [inline Exercises/1-random.js]:
```suggestion
  Math.floor(Math.random() * (superPuperMax - superPuperMin + 1)) + superPuperMin;
```

File: Exercises/1-random.js [hunk 3]
```diff
@@ -1,9 +1,8 @@
 'use strict';
 
-const random = (min, max) => {
-  // Generate random Number between from min to max
-  // Use Math.random() and Math.floor()
-  // See documentation at MDN
-};
+const random = (superPuperMin, superPuperMax) =>
+  // eslint-disable-next-line max-len
```
Comment (tshemsedinov) [inline Exercises/1-random.js]: Don't disable eslint rules

File: Exercises/4-methods.js [hunk 4]
```diff
@@ -1,21 +1,14 @@
...
-  //   m3(x, y, z) {
-  //     return [x, y, z];
-  //   }
-  // will return: [
-  //   ['m1', 1],
-  //   ['m2', 2],
-  //   ['m3', 3]
-  // ]
+  const sArr = [];
+  for (const i in iface) {
```
Comment (tshemsedinov) [inline Exercises/4-methods.js]: `i` is not a good naming, it's a string, so `name` or `key` will be ok

**Comment by Antiselekotin** [comment]
thanks for the comments

**Comment by tshemsedinov** [comment]
:+1:

**Pull request closed without merging**